stackit/provider.go

```diff
@@ -103,6 +103,7 @@ type providerModel struct {
 	ResourceManagerCustomEndpoint types.String `tfsdk:"resourcemanager_custom_endpoint"`
 	TokenCustomEndpoint           types.String `tfsdk:"token_custom_endpoint"`
 	JWKSCustomEndpoint            types.String `tfsdk:"jwks_custom_endpoint"`
+	EnableBeta                    types.Bool   `tfsdk:"enable_beta"`
```

Comment:
I think it would be a bit clearer if the name is more explicit
```suggestion
	EnableBetaResources                    types.Bool   `tfsdk:"enable_beta_resources"`
```